gc/base/ParallelDispatcher.hpp

```diff
@@ -46,6 +46,7 @@ class MM_ParallelDispatcher : public MM_Dispatcher
 	 * Data members
 	 */
 private:
+	uintptr_t _threadsToReserve;
```

Comment:
add a comment how is this used

Reply:
Added comment and addressed other suggestions